docs/USER_GUIDE.md

```diff
@@ -0,0 +1,328 @@
+---
+
+### 5.2. Szczegóły roli
+Szczegóły roli można rozwinąć, klikając w jakąkolwiek część jej pola. Kliknięcie innej roli spowoduje zwinięcie wcześniej rozwiniętej roli.
+
+Każda z ról dzielić się może na 3 sekcje: 
+- "Permitted Operations" zawiera operacje, które użytkownik może wykonać.
+- "Denied Operations" zawiera operacje, które użytkownik nie może wykonać.
+- "Subroles" zawiera podrole, po których dziedziczy dana rola.
+
+Kategorie nie są wyświetlane, jeżeli nie zawierają żadnych operacji.
+
+![Rola zawierająca wszystkie sekcje](images/user_guide/user_guide_23.png)
+![Rola zawierająca tylko sekcje "Permitted Operations"](images/user_guide/user_guide_24.png)
+
+Kliknięcie jednej z podroli w sekcji "Subroles" spowoduje rozwinięcie szczegółów tej podroli i przejście do niej.
+Pomiędzy kliknięciem a rozwinięciem podroli może nastąpić opóźnienie.
+
+Uprawnienia w kategoriach "Permitted Operations" oraz "Denied Operations" mogą być sortowane według namespace oraz typu zasobu.
+
+![Sortowanie uprawnień](images/user_guide/user_guide_25.png)
+
+Możliwe jest również filtrowanie uprawnień na podstawie operacji. 
+Po kliknięciu ikony po prawej stronie kolumny "Operations" zostanie wyświetlone pole do wybrania operacji. Po wybraniu zostaną wyświetlone tylko te uprawnienia, które zawierają wybraną operację.
+
+![Filtrowanie uprawnień](images/user_guide/user_guide_26.png)
+![Wyświetlenie tylko uprawnień z operacją "Update"](images/user_guide/user_guide_27.png)
+
+---
+
+### 5.3. Edycja ról
+Aby przejść do edycji roli, należy kliknąć przycisk "Edit Roles". Po kliknięciu zostaniemy przekierowani do formularza edycji ról.
+
+W formularzu edycji ról możliwe jest dodawanie nowych ról, usuwanie oraz edycja istniejących ról.
+
+Zmiany zapisać można klikając przycisk "Save". Aby porzucić zmiany, należy kliknąć przycisk "Cancel" i potwierdzić.
+
+![Formularz edycji ról](images/user_guide/user_guide_28.png)
+
+---
+
+#### Dodawanie, usuwanie i edycja ról
+
+Aby dodać nową rolę, należy kliknąć przycisk "Add Role". Po kliknięciu zostanie dodana nowa, pusta rola.
+Jeżeli rola nie posiada nazwy, nie zostanie zapisana.
+
+W ten sam sposób można dodawać nowe podrole. Wystarczy kliknąć "Add Subrole".
+
+![Dodawanie nowej roli](images/user_guide/user_guide_29.png)
+
+---
+
+#### Usuwanie ról
+
+Aby usunąć rolę, należy kliknąć czerwony przycisk "X" po prawej stronie nazwy roli.
+
+![Usuwanie roli](images/user_guide/user_guide_30.png)
+
+---
+
+#### Zmiana nazwy roli
+
+Aby zmienić nazwę roli, wystarczy kliknąć w pole tekstowe i wpisać nową nazwę.
+
+![Zmiana nazwy roli](images/user_guide/user_guide_31.png)
+
+---
+
+#### Dodawanie uprawnień do roli
+
+Aby dodać uprawnienie do roli, należy kliknąć przycisk "+" w sekcji "Permitted Operations" lub "Denied Operations". Po kliknięciu zostanie dodane nowe pole do wpisania uprawnienia.
+
+Domyślnie wprowadzone są puste pola. Odpowiada to przyznaniu wszystkich uprawnień dla wszystkich typów zasobów we wszystkich namespace'ach.
```

Comment:
Not true, nadal trzeba wybrać pozwolone akcje - delete, crete etc. To był mój patch

Reply:
Domyślnie wprowadzone są puste pola. Pusty typ zasobu oznacza wszystkie typy zasobów dostępne w aplikacji oraz "Helm". Pusta namespace oznacza wszystkie namespace'y. Pusta lista operacji oznacza brak uprawnień.